content/200-orm/500-reference/100-prisma-schema-reference.mdx

```diff
@@ -3148,6 +3188,41 @@ model User {
 </TabItem>
 </TabbedContent>
 
+### `ulid()`
+
+Generate a universally unique lexicographically sortable identifier based on the [ULID](https://github.com/ulid/spec) spec.
+
+#### Remarks
+
+- `ulid()` will produce 128-bit random identifier (e.g.: `01ARZ3NDEKTSV4RRFFQ69G5FAV`)
```

Comment:
Let's add that, when stored in string form, it requires 26 characters.

Reply:
```suggestion
- `ulid()` will produce 128-bit random identifier represented as a 26-character long alphanumeric string, e.g.: `01ARZ3NDEKTSV4RRFFQ69G5FAV`
```